quilt/test/test_install.py

```diff
@@ -75,11 +76,11 @@ def test_install_latest(self):
             file_contents = json.load(fd, object_hook=decode_node)
             assert file_contents == contents
 
-        with open('quilt_packages/objs/{hash}'.format(hash=table_hash)) as fd:
+        with open('quilt_packages/objs/{hash}'.format(hash=table_hash), 'rb') as fd:
```

Comment:
Instead of hard-coding the path here in the tests, use a PackageStore object to get the path.

Reply:
You've already updated it - I'll just have to merge and resolve conflicts.